src/test/java/com/imagination/cbs/service/helper/CreateBookingHelperTest.java

```diff
@@ -217,7 +209,18 @@ public void shouldCall_findByIdFromSiteOptionsRepository() {
 	}
 	
 	private BookingRequest createBookingRequest(){
-		
+		WorkDaysDto work = new WorkDaysDto();
```

Comment:
this method is too big... try to break in multiple private methods.

Reply:
changes done, please check